portfolio/src/main/java/com/google/sps/data/VideoAnalysis.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+import com.google.gson.Gson;
+import com.google.sps.data.PropertyNames;
+import com.google.sps.data.Query;
+import com.google.sps.data.SentimentTools;
+import com.google.sps.data.Video;
+
+// Represents a video analysis, holds the results from an analysis and can also analyze a video that is passed in.
+public class VideoAnalysis {
+    private static float sentimentScore;
+    private static float sentimentMagnitude;
+    private static String query;
+    private static int QUERY_SIZE = 7;
+    
+    //If given a video we will run analysis on it !!THIS IS SLOWER THAN JUST PASSING IN THE VALUES IF YOU HAVE THEM!!
+    public VideoAnalysis(Video video) {
+        SentimentTools sentimentAnalysis = new SentimentTools(video);
+        Query queryAnalysis = new Query(video, QUERY_SIZE);
+        this.sentimentScore = SentimentTools.getScore();
+        this.sentimentMagnitude = SentimentTools.getMagnitude();
+        this.query = queryAnalysis.toString();
+    }
+
+    //Pass in the results from a Video analysis to create this object
+    public VideoAnalysis(String _sentimentScore, String _sentimentMagnitude, String _query) {
+        this.sentimentScore = _sentimentScore;
+        this.sentimentMagnitude = _sentimentMagnitude;
```

Comment:
Since _sentimentScore and _sentimentMagnitue are Strings, they need to be parsed (e.g with Float.parseFloat).